vectorsearch/workload.json

```diff
@@ -6,23 +6,30 @@
         {
             "name": "{{ target_index_name }}",
             "body": "{{ target_index_body }}"
+        },
+        {
+            "name": "{{ train_index_name }}",
+            "body": "{{ train_index_body }}"
         }
     ],
     "corpora": [
     {
       "name": "cohere",
       "base-url": "https://dbyiw3u3rf9yr.cloudfront.net/corpora/vectorsearch/cohere-wikipedia-22-12-en-embeddings",
+      "target-index": "{{ target_index_name }}",
```

Comment:
Calling out here that this `target-index` param is not used anywhere in the workload, but it's necessary due to OSB validation. I'm not sure what the solution is, but I opened an [issue](https://github.com/opensearch-project/opensearch-benchmark/issues/572) about this.